src/styles/globals.css

```diff
@@ -45,6 +45,11 @@
   .dark ::-webkit-calendar-picker-indicator {
     filter: invert(1);
   }
+
+  .divide-y > :empty ~ :not([hidden]) {
```

Comment:
Is it possible to use just tailwind classes in place or create separate component for it?

Reply:
Short answer: `No`. See my comment about `:empty: hidden`